### PR TITLE
Improve delete multiple objects

### PR DESCRIPTION
**Pull request by qychrisyang**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Change Log
 All notable changes to QingStor SDK for Swift will be documented in this file.
 
+## [v2.0.1] - 2016-12-15
+### Changed
+- Improve: Computing Content-MD5 at DeleteMultipleObjectInput.
```
Comment (pengsrc) [inline CHANGELOG.md]: Improve the implementation of deleting multiple objects.

File: template/shared.tmpl [hunk 2]
```diff
@@ -141,7 +144,7 @@
 
 
 {{- define "RenderLocationProperties" -}}
-    {{- $customizedType := . -}}
+    {{- $customizedType := index . -}}
```
Comment (pengsrc) [inline template/shared.tmpl]: Missing the order number, should be `index $something $indexNumber`?

Comment (qychrisyang) [reply template/shared.tmpl]: Here is a mistake, has deleted the index

File: CHANGELOG.md (hunk 1 above)
Comment (qychrisyang) [reply CHANGELOG.md]: Has updated the description

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
 
+## [v2.0.1] - 2016-12-15
+### Changed
+- Improve the implementation of deleting multiple objects.
+
 ## [v2.0.0] - 2016-12-14
 ### Added
 - QingStor SDK for the Swift programming language.
 
 [Unreleased]: https://github.com/yunify/qingstor-sdk-swift/keep-a-changelog/compare/v2.0.0...HEAD
```
Comment (Xuanwo) [inline CHANGELOG.md:12]: Wrong url link, remove this line if there is no unreleased change.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
+[v2.0.1]: https://github.com/yunify/qingstor-sdk-go/compare/v2.0.0...v2.0.1
```
Comment (Xuanwo) [inline CHANGELOG.md:13]: The same mistake, s/go/swift

**Review by Xuanwo — CHANGES_REQUESTED** [review]

**Review by pengsrc — APPROVED** [review]

**Pull request merged**